source/_integrations/knx.markdown

```diff
@@ -852,7 +854,7 @@ move_long_address:
   required: false
   type: [string, list]
 move_short_address:
-  description: KNX group address for moving the cover stepwise up or down. Used by some covers also as the means to stop the cover, if no dedicated `stop_address` exists on the actuator. *DPT 1*
+  description: KNX group address for moving the cover stepwise up or down. Used by some covers also as the means to stop the cover. Note. Stepwise moves are only mapped to tilt angle functions in Home Assistant, as no stepwise move of cover position is generally supported by the architecture. If tilt angle is not supported, prefer the use of a `stop_address`. *DPT 1*
```

Comment:
We don't even use it to set tilt if there is a `angle_address` configured. So in most installations it is actually only used to as "stop". 

I think there should be a `:` instead of a `.` in "Note. "

Reply:
Yup, but there:in lies the grub. Ie by adding the addresses, we enable tilt features which may not exist in the hardware. So I like to push people to the stop address. Maybe we should have some other option, or make use of the device class.